crate/cli/src/actions/shared/unwrap_key.rs

```diff
@@ -91,12 +78,7 @@ impl UnwrapKeyAction {
             let key_bytes = general_purpose::STANDARD
                 .decode(b64)
                 .with_context(|| "failed decoding the unwrap key")?;
-            create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)
```

Comment:
I'd drop a § of comments to explain why unwrap is not symmetrical to wrap and why we removed the password option. Or where it fits best in code.

Reply:
right